dockerfiles/ci/centos/6/php-5.4/Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM datadog/dd-trace-ci:centos-6 as base
+
+ARG phpVersion
+ENV PHP_INSTALL_DIR_ZTS=${PHP_INSTALL_DIR}/${phpVersion}-zts
+ENV PHP_INSTALL_DIR_DEBUG_NTS=${PHP_INSTALL_DIR}/${phpVersion}-debug
+ENV PHP_INSTALL_DIR_NTS=${PHP_INSTALL_DIR}/${phpVersion}
+ENV PHP_VERSION=${phpVersion}
+
+# Download and extract PHP source
```

Comment:
I do love this idea, but unfortunately if I change `download-src.sh`, it will bust the layer caches for the base image (which is frozen in an EOL state) and I wouldn't be able to add any more layers to it. But I agree that we should change this for other containers that are not EOL.